<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DurakEkleMetro</class>
 <widget class="QDialog" name="DurakEkleMetro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Metro Durak Ekle</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0">
      <layout class="QGridLayout" name="gridLayout">
       <item row="7" column="2">
        <widget class="QSpinBox" name="spnOKDMetre">
         <property name="suffix">
          <string> m</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="QLineEdit" name="edtHatID"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="lblSemtAdi">
         <property name="text">
          <string>Semt Adı:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lblHatAdi">
         <property name="text">
          <string>Hat Adı: </string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblDurakID">
         <property name="text">
          <string>Durak ID:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QComboBox" name="cmbOKDAdi"/>
       </item>
       <item row="9" column="2">
        <widget class="QPushButton" name="pshIpt">
         <property name="text">
          <string>Iptal</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QLineEdit" name="edtDurakAdi"/>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QLabel" name="lblMDE">
         <property name="font">
          <font>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Metro Durak Ekle</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1" colspan="2">
        <widget class="QLineEdit" name="edtSemtAdi"/>
       </item>
       <item row="4" column="1" colspan="2">
        <widget class="QLineEdit" name="edtHatAdi"/>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="QLineEdit" name="edtDurakID"/>
       </item>
       <item row="9" column="1">
        <widget class="QPushButton" name="pshEkle">
         <property name="text">
          <string>Tamam</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblDurakAdi">
         <property name="text">
          <string>Durak Adı: </string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblHatID">
         <property name="text">
          <string>Hat ID:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QSpinBox" name="spnSKDMetre">
         <property name="suffix">
          <string> m</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QComboBox" name="cmbSKDAdi"/>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="lblKD">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Komşu Duraklar</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="lblOD">
         <property name="text">
          <string>Önceki Durak:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="lblSD">
         <property name="text">
          <string>Sonraki Durak:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
  <widget class="QStatusBar" name="statusbar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>edtDurakID</tabstop>
  <tabstop>edtHatID</tabstop>
  <tabstop>edtDurakAdi</tabstop>
  <tabstop>edtHatAdi</tabstop>
  <tabstop>edtSemtAdi</tabstop>
  <tabstop>cmbOKDAdi</tabstop>
  <tabstop>spnOKDMetre</tabstop>
  <tabstop>cmbSKDAdi</tabstop>
  <tabstop>spnSKDMetre</tabstop>
  <tabstop>pshEkle</tabstop>
  <tabstop>pshIpt</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
